Project.toml

```diff
@@ -18,6 +18,7 @@ Octavian = "6fd5a793-0b7e-452c-907f-f8bfe9c57db4"
 OpenBLAS_jll = "4536629a-c528-5b80-bd46-f80d51c5b363"
 ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+RecursiveFactorization = "f2c3362d-daeb-58d1-803e-2bc74f2840b4"
```

Comment:
Probably need a compat

Reply:
Added. I'm currently using my PR branch to RecursiveFactorization, which also depends on TriangularSolve.jl master and a local bug fix to Polyester.jl.